tools/Customization/DevHome.Customization/ViewModels/DevDriveInsights/OptimizeDevDriveDialogViewModel.cs

```diff
@@ -153,6 +153,50 @@ private void SetEnvironmentVariable(string variableName, string value)
         try
         {
             Environment.SetEnvironmentVariable(variableName, value, EnvironmentVariableTarget.User);
+
+            if (string.Equals(variableName, "CARGO_HOME", StringComparison.OrdinalIgnoreCase))

```

Comment:
Will this only ever apply to CARGO_HOME? If not (and maybe even if so, for readability) suggest refactoring this new code into a method that can be reused.

Reply:
This only applies to CARGO_HOME. But, yes, I will refactor this into a function.